docs/content/getting-started/basics.md

```diff
@@ -43,10 +66,10 @@ and execute them with the correct options for you.
 *You need to retrieve data from your production environment, but you don't want to
 have sensitive data on your local environment.*
 
-Let's say you have launched a `console db-tools:backup` on your production environment
+Let's say you have launched a <span class="standalone">`vendor/bin/db-tools backup`</span><span class="symfony">`console db-tools:backup`</span> on your production environment
```

Comment:
Most often, Symfony documents commands using `php bin/console foo:bar` and not simply `console foo:bar` - this way you can copy paste and be sure it'll work on any environment.

Reply:
damned

Reply:
done